config/config.yaml

```diff
@@ -318,105 +410,101 @@ clouds:
       # Hypershift Operator
       hypershift:
         image:
-          registry: quay.io
-          repository: acm-d/rhtap-hypershift-operator
           digest: sha256:0e6706e3bbc058a65f34dad7ccf048e56b1f32ca610c6f39f085ddb8bb1169ef
       # Backplane API
       backplaneAPI:
         image:
-          registry: arohcpsvcdev.azurecr.io
-          repository: app-sre/backplane-api
           digest: sha256:822477832a73c7eab7fe27200994f10030f708f4a752f33ded3f8f8eaa0470f6
       # Frontend
       frontend:
+        cert:
+          name: frontend-cert-{{ .ctx.regionShort }}
+          issuer: Self
         image:
-          registry: arohcpsvcdev.azurecr.io
-          repository: arohcpfrontend
           digest: '' # if empty uses commit sha of repo
       # Backend
       backend:
         image:
-          registry: arohcpsvcdev.azurecr.io
-          repository: arohcpbackend
           digest: '' # if empty uses commit sha of repo
+      # Image Sync
+      imageSync:
+        componentSync:
+          enabled: true
+          repositories: quay.io/redhat-user-workloads/maestro-rhtap-tenant/maestro/maestro,quay.io/acm-d/rhtap-hypershift-operator,quay.io/app-sre/uhc-clusters-service,quay.io/package-operator/package-operator-package,quay.io/package-operator/package-operator-manager,quay.io/package-operator/remote-phase-manager,quay.io/app-sre/backplane-api
+          secrets: 'quay.io:quay-io-bearer'
+          image:
+            digest: sha256:d838c4910bc53a5583dd501ed7e3ab08aa7c08b45b5997c90764c65ceef01a8f
+        ocMirror:
+          image:
+            digest: sha256:92dc2b18de0126caa2212f62c54023f6e8ecf12e2025c37a5f4151d0253ae14e
       # Shared SVC KV
       serviceKeyVault:
         name: 'aro-hcp-dev-svc-kv'
         rg: 'global'
         region: 'westus3'
-        private: false
-      # Management Cluster KVs
-      cxKeyVault:
-        softDelete: false
-        private: false
-      msiKeyVault:
-        softDelete: false
-        private: false
-      mgmtKeyVault:
-        softDelete: false
-        private: false
+        softDelete: true
       # disable soft delete on etcd KVs in DEV
       svc:
         subscription: ARO Hosted Control Planes (EA Subscription 1)
+        rg: hcp-underlay-{{ .ctx.regionShort }}-svc
```

Comment:
Good candidate for promotion up to top-level default?

Reply:
done